public/app/js/ui.js

```diff
@@ -18,28 +18,45 @@ onMainLoop(function(){
     var containerWidth = $("#text-message-input-area").width();
     var toolButtonTotalWidth = 0;
     if($("#text-message-input-area").hasClass("latex-editor-shown")) {
-        // Exclude the send button
-        $.each($(".message-tool-button").not("#send-message-button"), function(key, val){
-            toolButtonTotalWidth += $(val).outerWidth();
-        });
+        if($(window).width() > 732){
+            // Include all tool buttons
+            $.each($(".message-tool-button").not("#send-message-button").not("#equation-editor-button").not("#text-message-input-extra-symbols .latex-editor-symbol-button"), function(key, val){
+                toolButtonTotalWidth += $(val).outerWidth();
+            });
+        } else {
+            // Include all tool buttons
+            $.each($(".message-tool-button").not("#send-message-button").not("#text-message-input-extra-symbols .latex-editor-symbol-button"), function(key, val){
+                toolButtonTotalWidth += $(val).outerWidth();
+            });
+        }
     } else {
-        // Include all tool buttons
-        $.each($(".message-tool-button").not(".latex-editor-symbol-button"), function(key, val){
-            toolButtonTotalWidth += $(val).outerWidth();
-        });
+        if($(window).width() > 732){
+            // Include all tool buttons
+            $.each($(".message-tool-button").not(".latex-editor-symbol-button").not("#equation-editor-button"), function(key, val){
+                toolButtonTotalWidth += $(val).outerWidth();
+            });
+        } else {
+            // Include all tool buttons
+            $.each($(".message-tool-button").not(".latex-editor-symbol-button"), function(key, val){
+                toolButtonTotalWidth += $(val).outerWidth();
+                console.log(val);
```

Comment:
Don't forget to remove this.

![image](https://user-images.githubusercontent.com/31714259/36079643-68f76b9c-0f53-11e8-8a38-9dafb58d0cf6.png)